book/src/input/handling_input.md

```diff
@@ -0,0 +1,144 @@
+# Handling Input
+
+Amethyst uses an `InputHandler` to handle user input.
+You initialise this `InputHandler` by creating an `InputBundle` and adding it to the game data.
```

Comment:
I've found numerous instances of both `initialize` and `initialise` in the documentation, and it made me think - is it a goal for the Amethyst documentation to be consistent with the language used, or is it okay for the writer to choose their own preference?

Reply:
Good question I also noticed this, the problem is that some crates use the one and other crates the other, so it's hard to be consistent with this. We could change all the inconsistencies for now, but it will remain an ongoing problem.

Reply:
I think a PR that fixes all inconsistencies should also include an addition to the contribution guideline to keep to the selected language, and from then on, reviewers should also pay attention not to let different spellings - although I understand that can get tedious and difficult, especially for non-native speakers.